src/App.js

```diff
@@ -1,14 +1,78 @@
-import React from 'react';
+import React from 'react'
+import TodoForm from './components/TodoComponents/TodoForm'
+import TodoList from './components/TodoComponents/TodoList'
+import './components/TodoComponents/Todo.css'
+
+
+localStorage.getItem('state')
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props){
+    super(props);
+    this.state =
+      {Todos: 
+        [
+          {task: 'watch this week\'s The Orville',
+          id: 1528817077286,
+          completed: false},
+          {task: 'watch this week\'s Star Trek Discovery',
+          id: 1528817071031,
+          completed: false},
+        ],
+        Todo: '',
+      }
+  }
+
+  newTodo = (event) => this.setState({Todo: event.target.value})
+
+  addTask = (event) => {
+    event.preventDefault()
+    let tempStr = this.state.Todo
+    tempStr = tempStr.replace(/\s+/g, '')
+    if (tempStr != 0){
```

Comment:
tempStr will never equal 0 as it is a typeof string. You would probably want to get the length of tempStr and check to see if that is not equal to 0.